### PR TITLE
Update API v4 cdn_locks documentation to include currently missing response structure

### PR DESCRIPTION
**Pull request by Kasambx**
the following pull request provides some context for the deletion request of the API v4  

<hr/>

## Which Traffic Control components are affected by this PR?
- Documentation

## PR submission checklist
- [ ] This PR has tests
- [ ] This PR has documentation
- [ ] This PR has a CHANGELOG.md entry
- [ ] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://apache.org/security) for details)

### CODE REVIEW

File: docs/source/api/v4/cdn_locks.rst [hunk 1]
```diff
@@ -214,3 +214,13 @@ Response Structure
 		],
 		"lastUpdated": "2021-05-26T10:59:10-06"
 	}}
+	
+Response Structure
+-----------------
+:HTTPstatuscode: This would typically be a 204 No Content for a successful deletion, or a 4xx status code (such as 400 Bad Request, 401 Unauthorized, or 403 Forbidden) if the deletion was not successful.   	
+:Message:   This would provide additional information about the outcome of the request. For example, it may include an error message if the request failed, or a success message if the lock was deleted. 
+:Additionaldata: Depending on the specific implementation of the API, the response object may include additional data such as the ID of the deleted lock, the timestamp of when the lock was deleted, or other information relevant to the request.
+
```
Comment (ocket8888) [inline docs/source/api/v4/cdn_locks.rst:223]: Thank you for your contribution, but this section is not correct. It should be a a listing of the fields of the `response` property of the JSON object in the server's response body, just like those that appear under the "Response Structure" of the other methods for this endpoint.

**Review by ocket8888 — CHANGES_REQUESTED** [review]
Please fill out the Pull Request template to the best of your ability, and avoid deleting it entirely.

**Comment by mitchell852** [comment]
@Kasambx - are you able to make the requested change?

**Comment by ocket8888** [comment]
After a full year without activity, I'm closing this.

**Pull request closed without merging**